util/util_tls.go

```diff
@@ -76,20 +76,33 @@ func CreateSelfCert(
 			CommonName:   host,
 		},
 
-		IPAddresses: []net.IP{net.ParseIP(host)},
-		DNSNames:    []string{"localhost"},
-
 		NotBefore:          time.Now(),
 		NotAfter:           time.Now().AddDate(1, 0, 0),
 		SignatureAlgorithm: x509.SHA256WithRSA,
 
+		IsCA: true,
 		KeyUsage: x509.KeyUsageKeyEncipherment |
-			x509.KeyUsageDigitalSignature,
+			x509.KeyUsageDigitalSignature | x509.KeyUsageCertSign,
 
 		ExtKeyUsage:           []x509.ExtKeyUsage{x509.ExtKeyUsageServerAuth},
 		BasicConstraintsValid: true,
 	}
 
+	if ip := net.ParseIP(host); ip != nil {
+		tmpl.IPAddresses = append(tmpl.IPAddresses, ip)
+	} else {
+		ips, err := net.LookupIP(host)
+		if err != nil {
+			log.WithFields(log.Fields{
```

Comment:
Hi @vladimirvivien,

Actually, maybe log it as Debug level and that way if someone uses `REXRAY_DEBUG=true ./rexray install` they can see it, but otherwise not? What do you think?